internal/gengapic/example.go

```diff
@@ -225,9 +237,16 @@ func (g *generator) examplePagingCall(m *descriptor.MethodDescriptorProto) {
 	p("  }")
 	p("  // TODO: Use resp.")
 	p("  _ = resp")
+	p("")
+	p("  // TODO: Use the underlying response message.")
```

Comment:
I assume most people will never have to actually do this, I wonder if we should use more cautious language to imply they usually don't need this. Something like "If you need to access the underlying request you can do so by casting the Response as follows ". Any maybe even keep that line completely commented out. Thoughts?

Reply:
Sounds good to me. I agree, I think we should warn folks off a bit.

Reply:
I don't want to comment it out though b.c I want the import resolution for the reference.

Reply:
Updated. WDYT?